packages/kbn-optimizer/limits.yml

```diff
@@ -8,7 +8,7 @@ pageLoadAssetSize:
   charts: 195358
   cloud: 21076
   console: 46091
-  core: 432925
+  core: 468561
```

Comment:
35kb feels like a pretty dramatic raise considering the changes.

Reply:
@spalger I agree but I'm not sure where it's coming from.

Reply:
I suggest reading through https://www.elastic.co/guide/en/kibana/master/ci-metrics.html#ci-metric-resolving-overages to figure out what is causing it

Reply:
Oh, just noticing the comment you left earlier!

Reply:
Ah, the bloat comes from the `semver` import.
Running `node scripts/build_kibana_platform_plugins --dist --watch --focus core` shows that the import from `semver` increases the "page load bundle size" value from 197498 to 453561 and increases the bundle module count from 364 to 407.
@joshdover considering this change, should we rather just trust that we have a valid version format?